bot/discord/commands/ticket/remove.js

```diff
@@ -1,17 +1,18 @@
 exports.run = async (client, message, args) => {
-    if (!message.channel.name.includes('-ticket')) {
-        message.channel.send('This command is only to be used inside of ticket channels.')
-        return;
-    }
-
-    if (!args[1] || !message.guild.members.cache.get(args[1])) {
-        message.channel.send('Please run this command again with the users ID')
-    } else {
-        await message.channel.updateOverwrite(args[1], {
-            VIEW_CHANNEL: false,
-            SEND_MESSAGES: false,
-            READ_MESSAGE_HISTORY: false
-        })
-        message.channel.send("Success!")
-    }
+if (!message.channel.name.includes('-ticket')) {
+    message.channel.send('This command is only to be used inside of ticket channels.')
+    return;
+}
+let member = message.mentions.members.first() || message.guild.members.cache.get(args[0]);
```

Comment:
it needs to be `message.guild.members.cache.get(args[1])` as the command is `DBH!ticket remove {id}` `args[0]` would be the word "remove" and the id would be `args[1]`

Reply:
Fixed